konrad/radiation/rrtmg.py

```diff
@@ -80,7 +95,8 @@ def init_radiative_state(self, atmosphere, surface):
             cloud_optical_properties=self._cloud_optical_properties,
             cloud_ice_properties=self._cloud_ice_properties,
             cloud_liquid_water_properties='radius_dependent_absorption',
-            cloud_overlap_method=overlap,
+            cloud_overlap_method='maximum_random',
```

Comment:
Sorry, I guess this was mine, but I think we should stick to the old version, ``cloud_overlap_method=overlap`` with ``overlap`` depending on whether mcica is being used or not.